modules/saku-policy-store/src/saku/dal_datalevin.clj

```diff
@@ -17,28 +17,17 @@
        drns))
 
 
-(defn ^:private impl-get-resource-policies [db drns]
+(defn ^:private impl-get-*-policies [db drns statement-attr]
```

Comment:
Parameterize the query by `statement-attr` to allow reuse between resource and identity policies.